_rules/html-has-lang-b5c3f8.md

```diff
@@ -86,46 +86,6 @@ The `xml:lang` attribute specified has a non-empty value. The rule expects a non
 <html xml:lang="en" lang=""></html>
 ```
 
-#### Passed Example 6
```

Comment:
I would be in favour of adding a note explicitly saying that the rule checks for presence, not validity of the value.
(cutting down the number of useless examples *is* good :-D)